### PR TITLE
Forms design updates

### PR DESCRIPTION
**Pull request by josemarques**
screen shot 2018-10-31 at 17 06 24

Product edit:
- Adds card with the title and description fields in the  

screen shot 2018-10-31 at 17 07 33

Settings:
- Adds cards to match Calypso's cards patterns
- Updates typography in titles and description and makes it a header section

### CODE REVIEW

File: assets/css/calypsoify.css [hunk 1]
```diff
@@ -316,6 +316,105 @@ table.widefat {
 	color: #2E4453;
 }
 
+/* Forms */
+
+#post-body-content {
+	background-color: #fff;
+	box-shadow: 0 0 0 1px rgba(200,215,225,0.5), 0 1px 2px #e9eff3;
```
Comment (joshuatf) [inline assets/css/calypsoify.css]: Can we simplify this to a `border: 1px solid #hex` here?  It appears to be a single solid border around its edge, but I may be missing some details here.

File: assets/css/calypsoify.css [hunk 2]
```diff
@@ -316,6 +316,105 @@ table.widefat {
...
+#wp-content-editor-tools {
+	background: #fff;
+}
+
+/* Adds card layout style to forms */
+
+#mainform h2,
+#mainform h3 {
+	background-color: #fff;
+	border: 1px solid rgba(200,215,225,0.5);
```
Comment (joshuatf) [inline assets/css/calypsoify.css]: Do these need to be rgba or can they be hex?

File: assets/css/calypsoify.css [hunk 3]
```diff
@@ -316,6 +316,105 @@ table.widefat {
...
+
+#post-body-content {
+	background-color: #fff;
+	box-shadow: 0 0 0 1px rgba(200,215,225,0.5), 0 1px 2px #e9eff3;
+	box-sizing: border-box;
+	padding: 24px;
+}
+
+#title-prompt-text {
+	clip: initial !important; /*avoid clipping when field is on focus */
```
Comment (joshuatf) [inline assets/css/calypsoify.css]: Nice work on the clipping property!  If you change the selector here to `#titlediv #title-prompt-text` we can probably remove all of those important tags unless you saw something else overriding these properties.

File: assets/css/calypsoify.css (hunk 2 above)
Comment (josemarques) [reply assets/css/calypsoify.css]: I copied the value from .nav-tab-wrapper. But we can use #e4eaef.

File: assets/css/calypsoify.css (hunk 3 above)
Comment (josemarques) [reply assets/css/calypsoify.css]: Will do.

**Comment by josemarques** [comment]
@joshuatf were these changes squashed? I'm not seeing the update on the forms on master.

**Pull request merged**